dist/js/apps/filter-page/app.js

```diff
@@ -248,13 +248,13 @@
 		/* Private */
 
 		var cycleDisplay = function cycleDisplay() {
-			var $resultsDisplay = angular.element('#results-display');
-			$animate.addClass($resultsDisplay, 'fade-out');
+			var resultsDisplayElement = document.getElementById('results-display');
```

Comment:
I actually didn't have a problem with the existing code, if the case is that jQuery is unavoidable with the event and/or animation. I am just trying to stick to this [philosophy](https://docs.angularjs.org/misc/faq#dom-manipulation).

Saying that I don't have a good handle on the animation in angular, so not even what the angular way would be. 


This refactored code looks good though.